modules/web/web-api/src/main/java/com/enonic/xp/web/vhost/VirtualHostService.java

```diff
@@ -0,0 +1,18 @@
+package com.enonic.xp.web.vhost;
+
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+
+public interface VirtualHostService
+{
+
+    boolean isEnabled();
+
+    VirtualHost getVirtualHost( final String name );
+
+    List<VirtualHost> getVirtualHosts();
+
+    VirtualHost resolve( final HttpServletRequest req );
```

Comment:
isEnabled, getVirtualHost, getVirtualHosts should be in a different service (VirtualHostConfigService)
resolve should stay here.

Reply:
done